stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/from_stix_map.json

```diff
@@ -46,7 +46,7 @@
       "pid": ["processes.processId", "processes.parentProcessId", "registryKeyStates.processId"],
       "created_time": ["processes.createdDateTime"],
       "parent_ref.pid": ["processes.parentProcessId"],
-      "image_ref.path": ["processes.path"]
+      "image_ref.parent_directory.path": ["processes.path"]
```

Comment:
Should be `"image_ref.parent_directory_ref.path"`

Reply:
done

Reply:
It's still missing the `_ref` in `parent_directory_ref`